### PR TITLE
Contacts feature added and Fixes in selectors

### PR DESCRIPTION
**Pull request by EscarlethFatima**
Missing comments

### CODE REVIEW

File: test/utils/CommonActions.js [hunk 1]
```diff
@@ -145,48 +126,6 @@ class CommonActions {
         return this.getElement(elementCss).getText().toString() === textToVerify;
     }
 
-    /**
-     * @param elementCss auto completer textField.
-     * @param textToSelect select text on autoCompleter.
-     * @param autoCompleterKey locator to auto completer button on classic theme.
-     */
-    static selectOnAutoCompleterTextField(elementCss, textToSelect, autoCompleterKey) {
```
Comment (omar-limbert) [inline test/utils/CommonActions.js:153]: Are you sure? this methods is to AutoCompleter Field on Classic and Lightning theme, maybe you need think this.

File: test/utils/CommonActions.js [hunk 2]
```diff
@@ -145,48 +126,6 @@ class CommonActions {
...
-            this.autoCompleterLightTheme(textToSelect, autoCompleterKey);
-        }
-
-    }
-
-
-    /**
-     * @param textToSelect select text on autoCompleter.
-     */
-    static autoCompleterClassicTheme(textToSelect) {
```
Comment (omar-limbert) [inline test/utils/CommonActions.js:169]: Are you sure? this methods is to AutoCompleter Field on Classic and Lightning theme, maybe you need think this.

File: test/utils/CommonActions.js [hunk 3]
```diff
@@ -145,48 +126,6 @@ class CommonActions {
...
-        browser.pause(3000);
-        this.clickWebElement(css);
-        browser.switchTab(handle[0]);
-    }
-
-    /**
-     * @param textToSelect select text on autoCompleter.
-     * NEED IMPROVE
-     */
-    static autoCompleterLightTheme(textToSelect, autoCompleterKey) {
```
Comment (omar-limbert) [inline test/utils/CommonActions.js:185]: Are you sure? this methods is to AutoCompleter Field on Classic and Lightning theme, maybe you need think this.

File: test/specs/accounts/update.account.spec.js [hunk 4]
```diff
@@ -1,57 +1,40 @@
...
 describe('Acceptance Test to Modify an Account', function () {
-    this.retries(3);
 
-    let accountModifiedData = {
-        name: 'Account Updated',
+    let accountModified = {
+        name: 'Account Sample',
         number: 100,
         site: 2,
-        parent: 'Edge Communications',
```
Comment (omar-limbert) [inline test/specs/accounts/update.account.spec.js:15]: Maybe you need use this to test AutoCompleter Method.

File: test/pages/section.page.js [hunk 5]
```diff
@@ -34,13 +37,6 @@ class Section {
         commonActions.pauseInSeconds(3);
         let finalSelector = commonActions.getSelector(section);
         commonActions.clickWebElement(finalSelector);
-        if (isClassic) {
-            commonActions.closePopMessage(this.lightningMessage.message, this.lightningMessage.closeButton);
```
Comment (omar-limbert) [inline test/pages/section.page.js:38]: Are you sure? this line is to close Spam message after choose any section.

File: test/utils/CommonActions.js [hunk 6]
```diff
@@ -127,16 +118,6 @@ class CommonActions {
         }
     }
 
-    /**
-     * @param message popup message.
-     * @param closeButton close button on popup message at Classic Theme.
-     */
-    static closePopMessage(message, closeButton) {
```
Comment (omar-limbert) [inline test/utils/CommonActions.js:134]: Are you sure? this method is to close Spam message after choose any section on classic theme.

File: test/utils/CommonActions.js [hunk 7]
```diff
@@ -52,14 +51,6 @@ class CommonActions {
 
     }
 
-    /**
-     * Method to wait element has been enable.
-     * @param elementCSS WebElement locator.
-     */
-    static waitElementToReady(elementCSS) {
```
Comment (omar-limbert) [inline test/utils/CommonActions.js:59]: Maybe you need check this is to improve stability.

File: test/specs/accounts/create.account.spec.js [hunk 8]
```diff
@@ -1,54 +1,40 @@
...
+        name: 'Account Required'
     };
 
     // Login application.
     beforeEach(function () {
-        loginPage.login(config.username, config.password);
-    });
-
-    // Delete an accounts created.
-    afterEach(function () {
```
Comment (omar-limbert) [inline test/specs/accounts/create.account.spec.js:35]: maybe you will need this afterEach to delete account hast been created before.

File: test/specs/contacts/create.contact.spec.js [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+    };
+    let contactWithRequiredField = {
+        lastName: 'TheContact'
+    };
+
+    // Login application.
+    beforeEach(function () {
+        login.login(loginApplication.username, loginApplication.password);
+    });
+
```
Comment (omar-limbert) [inline test/specs/contacts/create.contact.spec.js:25]: After each missing.

File: test/specs/accounts/update.account.spec.js [hunk 10]
```diff
@@ -1,57 +1,40 @@
...
-        // Create new account to modify.
+        loginPage.login(loginApplication.username, loginApplication.password);
         topSideBar.goToSection('accounts');
         content.clickOnNewButton();
-        form.fillAccountWithRequiredFieldsAndSave(accountToModify);
-
-    });
-
-    // Delete an accounts created.
-    afterEach(function () {
```
Comment (omar-limbert) [inline test/specs/accounts/update.account.spec.js:42]: maybe you will need this afterEach to delete account hast been created before.

**Review by omar-limbert — COMMENTED** [review]
Please check this comments.

**Comment by carledriss** [comment]
@Escarleth 
Please fix Omar’s observations

File: test/utils/CommonActions.js (hunk 6 above)
Comment (EscarlethFatima) [reply test/utils/CommonActions.js:134]: sure it is not neccesary.I tried it

File: test/utils/CommonActions.js (hunk 7 above)
Comment (EscarlethFatima) [reply test/utils/CommonActions.js:59]: sure it is not neccesary.I tried it

File: test/specs/accounts/create.account.spec.js (hunk 8 above)
Comment (EscarlethFatima) [reply test/specs/accounts/create.account.spec.js:35]: I am working on this

File: test/pages/section.page.js (hunk 5 above)
Comment (EscarlethFatima) [reply test/pages/section.page.js:38]: sure it is not neccesary.I tried it

File: test/specs/contacts/create.contact.spec.js (hunk 9 above)
Comment (EscarlethFatima) [reply test/specs/contacts/create.contact.spec.js:25]: I am working on this

File: test/specs/accounts/update.account.spec.js (hunk 10 above)
Comment (EscarlethFatima) [reply test/specs/accounts/update.account.spec.js:42]: the same as above

File: test/specs/accounts/update.account.spec.js (hunk 4 above)
Comment (EscarlethFatima) [reply test/specs/accounts/update.account.spec.js:15]: To run in lightning and classic I decided to remove some, but I will add again for classic

File: test/utils/CommonActions.js (hunk 3 above)
Comment (EscarlethFatima) [reply test/utils/CommonActions.js:185]: To run in lightning and classic I decided to remove some, but I will add again for classic

File: test/utils/CommonActions.js (hunk 1 above)
Comment (EscarlethFatima) [reply test/utils/CommonActions.js:153]: To run in lightning and classic I decided to remove some, but I will add again for classic

File: test/utils/CommonActions.js (hunk 2 above)
Comment (EscarlethFatima) [reply test/utils/CommonActions.js:169]: the same as above

**Pull request merged**